plugins/cpp/test/src/cppparsertest.cpp

```diff
@@ -135,10 +140,12 @@ TEST_F(CppParserTest, FunctionDeclarationOnly)
 TEST_F(CppParserTest, FunctionWithMultipleDeclarations)
 {
   _transaction([&, this]() {
-    model::CppFunction callee = _db->query_value<model::CppFunction>(
+    RCppFunction callees = _db->query<model::CppFunction>(
       QCppFunction::name == "multiFunction");
+    EXPECT_GT(callees.size(), 0);
```

Comment:
Here the `multiFunction` is declared 2 times and defined 1 time, so all together 3 records in `CppFunction` will be found.